tooling/components/io.siddhi.distribution.editor.core/src/main/java/io/siddhi/distribution/editor/core/util/SourceEditorUtils.java

```diff
@@ -475,6 +723,54 @@ private static ProcessorMetaData generateProcessorMetaData(Class<?> processorCla
                 processorMetaData.setParameters(parameterMetaDataList);
             }
 
+            List<SyntaxMetaData> syntaxList = new ArrayList<>();
+            String finalClipBoardSyntax = "";
+            String finalSyntax = "";
+            StringBuilder syntax = new StringBuilder();
+            StringBuilder clipBoardSyntax = new StringBuilder();
+              /* Generate syntax annotation data for function,attribute aggregation,windows,stream
+            function and stream */
+            if (Constants.FUNCTION_EXECUTOR.equalsIgnoreCase(processorType)
+                    || Constants.ATTRIBUTE_AGGREGATOR.equalsIgnoreCase(processorType) ||
+                    Constants.WINDOW_PROCESSOR.equalsIgnoreCase(processorType)
+                    || Constants.STREAM_FUNCTION_PROCESSOR.equalsIgnoreCase(processorType) ||
+                    Constants.STREAM_PROCESSOR.equalsIgnoreCase(processorType)) {
+                processorMetaData.setSyntax(windowFunctionSyntaxGeneration(extensionAnnotation,
+                        parameterMap, syntaxList, finalClipBoardSyntax, finalSyntax, syntax, clipBoardSyntax));
+            }
+
+            /*Generate syntax annotation data for source and sink*/
+            if ((Constants.SOURCE.equals(processorType))
+                    || (Constants.SINK.equals(processorType))) {
+                processorMetaData.setSyntax(sourceSinkSyntaxGeneration(extensionAnnotation,
+                        parameterMap, syntaxList, finalClipBoardSyntax, finalSyntax, syntax, clipBoardSyntax));
+            }
+
+/*            Generate Syntax annotation data for source mapper and sink mapper*/
```

Comment:
Format add // comments if they are only one line

Reply:
acf2589 this commit fix this change